Valghalla.Internal.Web/src/assets/i18n/da.json

```diff
@@ -308,7 +308,7 @@
         "export_to_excel": "Eksportér som Excel-fil",
         "export_to_csv": "Eksportér som CSV-fil",
         "select_columns": "Vælg kolonner",
-        "full_name": "Fuldt navn",
+        "full_name": "Fulde navn",
```

Comment:
Can this change affect somewhere else in the system? Maybe better to have seperate strings only for the export?